### PR TITLE
chore(core): Dismissione WISP: Migrazione a Lotti  [PAGOPA-2039]

### PR DESCRIPTION
**Pull request by jacopocarlini**
### List of changes

🗑️ deleted named values `ci_whitelist` and `stations_whitelist`
🖼️ edited fragment `decoupler-algorithm`: template variables are added to replace named values
🧹 `pre-commit` auto changes

🔴 Important! **Before** apply `core` then `nodo-app` domain

```
cd core
sh terraform.sh apply prod -target=azapi_resource.decoupler_algorithm

cd /domains/nodo-app
sh terraform.sh apply weu-prod -target=azurerm_api_management_named_value.wisp_ci_whitelist_named_value -target=azurerm_api_management_named_value.wisp_station_whitelist_named_value --auto-approve
```

### Motivation and context

[Jira PAGOPA-2039](https://pagopa.atlassian.net/browse/PAGOPA-2039)

### Type of changes

- [ ] Add new resources
- [x] Update configuration to existing resources
- [x] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [x] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [x] No

### Other information

---

### CODE REVIEW

File: src/domains/nodo-app/04_apim_wisp_dismantling_policies.tf [hunk 1]
```diff
@@ -25,22 +25,6 @@ resource "azurerm_api_management_named_value" "wisp_channel_whitelist_named_valu
   value               = var.wisp_converter.channel_whitelist
 }
 
-resource "azurerm_api_management_named_value" "wisp_station_whitelist_named_value" {
```
Comment (AngeloCaporaso) [inline src/domains/nodo-app/04_apim_wisp_dismantling_policies.tf:28]: should be kept until the final transition to the prod environment, no?

**Review by AngeloCaporaso — CHANGES_REQUESTED** [review]
I prefer the `apply` commands to be annotated to be clear about the actions to be taken in the switch from named-values (_as-is_) to variables (_to-be_)

File: src/domains/nodo-app/04_apim_wisp_dismantling_policies.tf (hunk 1 above)
Comment (jacopocarlini) [reply src/domains/nodo-app/04_apim_wisp_dismantling_policies.tf:28]: Yes, I added this important consideration in the description of the PR

**Comment by aomegax** [comment]
replaced by #2382

**Pull request closed without merging**